charts/deepgram-self-hosted/templates/license-proxy/license-proxy.deployment.yaml

```
@@ -74,8 +107,13 @@ spec:
             memory: "{{ .Values.licenseProxy.resources.limits.memory }}"
             cpu: "{{ .Values.licenseProxy.resources.limits.cpu }}"
         volumeMounts:
-        - name: license-proxy-config-volume
+        {{- if .Values.licenseProxy.upstream.useBilling }}
+        - name: working-config
+          mountPath: /etc/config
+        {{- else }}
+        - name: configmap-volume
           mountPath: /etc/config
+        {{- end }}
```

```suggestion
        - name: {{- if .Values.licenseProxy.upstream.useBilling -}} working-config {{- else -}} configmap-volume  {{- end }}
          mountPath: /etc/config
```

Nit, you could collapse this and make it more obviously readable what is different between the two. (this applies to all three deployments where we do this)

---

Thanks, I collapsed logic in those spots, along with related volume cleanup: https://github.com/deepgram/self-hosted-resources/pull/110/commits/2ac760fa443389a955ef8e1f541059218bca7a25